generators/Exercises/IsbnVerifier.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Generators.Input;
+
+namespace Generators.Exercises
+{
+    public class IsbnVerifier : Exercise
+    {
+        protected override void UpdateCanonicalData(CanonicalData canonicalData)
+        {
+            canonicalData.Exercise = "Isbn";
+            foreach (var canonicalDataCase in canonicalData.Cases)
+            {
+                canonicalDataCase.Property = "IsValid";
```

Comment:
Much better than the existing property! In fact, I've gone and submitted a [PR](https://github.com/exercism/problem-specifications/pull/999) to change the canonical data.